app/static/js/createEvents.js

```diff
@@ -103,20 +154,78 @@ $(document).ready(function () {
+    if(!(document.getElementById('inputEventName').value === '')){
+      document.getElementById('eventName').value = document.getElementById('inputEventName').value; //keeps main page event name for multiple event modal
+    }
+    var recurringStatus = $("input[id='checkIsRecurring']:checked").val(); //this line function is to retrive ON when its toggle for recurring event on createEvent.html line 158
+    var multipleOfferingStatus = $("input[id='checkIsMultipleOffering']:checked").val();// this line function is to retrive ON when toggle for multiple offering event button createEvent.html line 160
+    modalOpenedByEditButton = ($(this).attr('id') === 'edit_modal');
+    if (multipleOfferingStatus == 'on' && recurringStatus == 'on'){
+      console.log("Both recurring and multiple offering are on. Showing message...");
+      msgFlash("You may not toggle recurring event and multiple time offering event at the same time!", "danger");
+      $(event.target).prop('checked', false);
+      return; 
+    }
+    
     if (recurringStatus == 'on') {
       $(".endDateStyle, #recurringTableDiv").removeClass('d-none');
+      $("#checkIsMultipleOffering").prop('checked', false);
+      $('#multipleOfferingTableDiv').addClass('d-none');
       $(".endDatePicker").prop('required', true);
-    } else {
-      $(".endDateStyle, #recurringTableDiv").addClass('d-none');
+    } 
+    else{
+      $(".endDateStyle, #recurringTableDiv").addClass('d-none');// this line add the display none button of bootstrap so that the end-date div disappears for recurring event
       $(".endDatePicker").prop('required', false);
     }
+    if (multipleOfferingStatus == 'on') {
+      $(".startDatePicker").prop('required', false);
+      $("#multipleOfferingTableDiv").removeClass('d-none');
+      $("#checkIsRecurring").prop('checked', false);
+      $(".endDateStyle, #recurringTableDiv").addClass('d-none');
+      $('#modalMultipleOffering').modal('show');
+      $('#nonMultipleOfferingTime, #nonMultipleOfferingDate').addClass('d-none'); // this line hides the non multiple offering time and dates and replace them with recurring table div for multiple offering events to show
+    }
+    else{
+      $("#multipleOfferingTableDiv").addClass('d-none');
+      $('#modalMultipleOffering').modal('hide');
+      $('#nonMultipleOfferingTime, #nonMultipleOfferingDate').removeClass('d-none');
+      $(".startDatePicker").prop('required', true);
+    }
+  });
+  
+  $("#cancelModalPreview, #multipleOfferingXbutton").click(function(){ //this function is to untoggle the button when the modal has cancel or close button being clicked
+    if (modalOpenedByEditButton == false) {
+    $("#checkIsMultipleOffering").prop('checked', false);
+    $('#nonMultipleOfferingTime, #nonMultipleOfferingDate').removeClass('d-none');
+    $("#multipleOfferingTableDiv").addClass('d-none');
+    $('#modalMultipleOffering').modal('hide');
+    $('.extraSlots').children().not(':first').remove();
+  }
+  });
```

Comment:
Same deal with the end-of-line comments in the above chunk of code.

Reply:
corrected commenting